compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -201,29 +373,58 @@ struct QuantizeDequantizeWeights final : public luci::CircleNodeMutableVisitor<b
       {
         auto circle_const = loco::must_cast<luci::CircleConst *>(circle_node);
 
+        // Find min/max per channel-wise
+        if (granularity)
```

Comment:
Can you clarify the condition? E.g., granularity == QuantizationGranularity::ChannelWise